projects/ngrx.io/content/guide/store/index.md

```diff
@@ -13,6 +13,13 @@ Store is RxJS powered state management for Angular applications, inspired by Red
 
 Detailed installation instructions can be found on the [Installation](guide/store/install) page.
 
+## Diagram
+
+The following diagram represents the overall general flow of application state in ngrx. 
```

Comment:
Nits: use NgRx when referring to the platform

Reply:
Just pushed fix